test/test_torch.py

```diff
@@ -14181,7 +14181,7 @@ def test_blas_alpha_beta_empty(self, device, dtype):
                          torch.addmv(input=input, mat=mat, vec=vec, alpha=alpha, beta=beta, out=out))
 
         # TODO: update this once torch.addmm is supported for complex
-        if dtype.is_complex:
+        if dtype.is_complex and device != 'cpu':
```

Comment:
cc @anjali411

Reply:
addmm currently works for complex on CUDA. will enable matmul for complex on cpu in a follow-up PR